ls1intum/Artemis
# Fix: Text JPlag Export has "null" in Zipname

### Checklist

### Description
- Text Exercises do not have a short name.
- Use Exercise Title instead.

### Steps for Testing
1. Download JPlag Report for a Text Exercise
2. Check if Exercise Name is in Filename of the downloaded ZIP file.
